corehq/apps/analytics/ab_tests.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 import random
+import logging
 from django.core.cache import cache
 
 from memoized import memoized
 from corehq.apps.accounting.models import Subscription
 
 
+logger = logging.getLogger('')
```

Comment:
I hear it's more typical to pass `__name__` rather than blank string. Conversation on this here: https://github.com/dimagi/commcare-hq/pull/22287#discussion_r231175201